README.md

```diff
@@ -260,95 +305,99 @@ Instead of using column text (that sometime re-order the data), force an index a
 ```
 
 ### countDuplicateHeadings
-Default is 'true'. If set to 'false' duplicate headings will not get a trailing _<NUMBER>. The value of 
-the field will be the last value found in the table row:
 
-```
-// Table
-| PLACE | VALUE | PLACE | VALUE |
-|   abc |     1 |   def |     2 |
-|   ghi |     3 |   jkl |     4 |
+Default is `true`. If set to `false`, duplicate headings will not get a trailing
+number. The value of the field will be the last value found in the table row:
+
+PLACE | VALUE | PLACE | VALUE
+------|-------|-------|------
+  abc |     1 |   def |     2
+  ghi |     3 |   jkl |     4
 
+```js
 // Example output with countDuplicateHeadings:false
 [
     {
         PLACE: 'def', VALUE: '2'
     },
-    {    
+    {
         PLACE: 'jkl', VALUE: '4'
     }
 ]
 ```
 
 ### ignoreColumns
+
 Array of indexes to be ignored, starting with 0. Default is 'null/undefined'.
 
-```
-// Table
-| NAME | PLACE | WEIGHT | SEX | AGE |
-|  Mel |     1 |     58 |   W |  23 |
-|  Tom |     2 |     78 |   M |  54 |
-| Bill |     3 |     92 |   M |  31 |
+ NAME | PLACE | WEIGHT | SEX | AGE
+------|-------|--------|-----|----
+ Mel  |     1 |     58 |   W |  23
+ Tom  |     2 |     78 |   M |  54
+ Bill |     3 |     92 |   M |  31
 
+```js
 // Example output with ignoreColumns: [2, 3]
 [
     {
         NAME: 'Mel', PLACE: '1', AGE: '23'
     },
-    {    
+    {
         NAME: 'Tom', PLACE: '2', AGE: '54'
     },
-    {    
+    {
         NAME: 'Bill', PLACE: '3', AGE: '31'
     }
 ]
 ```
 
 ### onlyColumns
+
 Array of indexes that are taken, starting with 0. Default is 'null/undefined'.
 If given, this option overrides ignoreColumns.
 
-```
-// Table
-| NAME | PLACE | WEIGHT | SEX | AGE |
-|  Mel |     1 |     58 |   W |  23 |
-|  Tom |     2 |     78 |   M |  54 |
-| Bill |     3 |     92 |   M |  31 |
+ NAME | PLACE | WEIGHT | SEX | AGE
+------|-------|--------|-----|----
+ Mel  |     1 |     58 |   W |  23
+ Tom  |     2 |     78 |   M |  54
+ Bill |     3 |     92 |   M |  31
 
+```js
 // Example output with onlyColumns: [0, 4]
 [
     {
         NAME: 'Mel', AGE: '23'
     },
-    {    
+    {
         NAME: 'Tom', AGE: '54'
     },
-    {    
+    {
         NAME: 'Bill', AGE: '31'
     }
 ]
 ```
 
 ### ignoreHiddenRows
+
 Indicates if hidden rows (display:none) are ignored. Default is true:
 
-```
-// Table
- | NAME | PLACE | WEIGHT | SEX | AGE |
- |  Mel |     1 |     58 |   W |  23 |
- |  Tom |     2 |     78 |   M |  54 |
- | Bill |     3 |     92 |   M |  31 |
-*|  Cat |     4 |      4 |   W |   2 |*
+ NAME | PLACE | WEIGHT | SEX | AGE
```

Comment:
Structure looks better than before :-) !!

Reply:
Glad you like it :-)